diff -urN kdepim-4.8.97/akregator/configuration/akregator_config_advanced.desktop kdepim-4.8.97-patched/akregator/configuration/akregator_config_advanced.desktop
--- kdepim-4.8.97/akregator/configuration/akregator_config_advanced.desktop	2012-07-11 09:44:13.000000000 +1100
+++ kdepim-4.8.97-patched/akregator/configuration/akregator_config_advanced.desktop	2012-07-16 21:46:55.748768010 +1100
@@ -132,6 +132,7 @@
 X-KDE-Keywords[nl]=akregator, instellen, instellingen, geavanceerd
 X-KDE-Keywords[pl]=akregator, konfiguruj, ustawienia, zaawansowane
 X-KDE-Keywords[pt]=akregator, configurar, configuração, avançado
+X-KDE-Keywords[ru]=akregator, configure, settings, advanced, настроить, сконфигурировать, настройки, дополнительные
 X-KDE-Keywords[pt_BR]=akregator,configurar,configurações,avançado
 X-KDE-Keywords[sk]=akregator, nastaviť, nastavenia, pokročilé
 X-KDE-Keywords[sr]=akregator,configure,settings,advanced,Акрегатор,подешавање,поставке,напредно
diff -urN kdepim-4.8.97/akregator/configuration/akregator_config_appearance.desktop kdepim-4.8.97-patched/akregator/configuration/akregator_config_appearance.desktop
--- kdepim-4.8.97/akregator/configuration/akregator_config_appearance.desktop	2012-07-11 09:44:13.000000000 +1100
+++ kdepim-4.8.97-patched/akregator/configuration/akregator_config_appearance.desktop	2012-07-16 21:46:55.748768010 +1100
@@ -150,6 +150,7 @@
 X-KDE-Keywords[nl]=akregator, instellen, instellingen, uiterlijk, look, feel
 X-KDE-Keywords[pl]=akregator, konfiguruj, ustawienia, wygląd, odczucia
 X-KDE-Keywords[pt]=akregator, configurar, configuração, aparência, comportamento
+X-KDE-Keywords[ru]=akregator,configure,settings,appearance,look,feel,настроить,сконфигурировать,внешний вид,вид,поведение
 X-KDE-Keywords[pt_BR]=akregator,configurar,configurações,aparência,comportamento
 X-KDE-Keywords[sk]=akregator, nastaviť, nastavenia, vzhľad, pohľad, cítenie
 X-KDE-Keywords[sr]=akregator,configure,settings,appearance,look,feel,Акрегатор,подешавање,поставке,изглед,осећај
diff -urN kdepim-4.8.97/akregator/configuration/akregator_config_archive.desktop kdepim-4.8.97-patched/akregator/configuration/akregator_config_archive.desktop
--- kdepim-4.8.97/akregator/configuration/akregator_config_archive.desktop	2012-07-11 09:44:13.000000000 +1100
+++ kdepim-4.8.97-patched/akregator/configuration/akregator_config_archive.desktop	2012-07-16 21:46:55.755768011 +1100
@@ -130,6 +130,7 @@
 X-KDE-Keywords[nl]=akregator, instellen, instellingen, archief
 X-KDE-Keywords[pl]=akregator, konfiguruj, ustawienia, archiwum
 X-KDE-Keywords[pt]=akregator, configurar, configuração, arquivo
+X-KDE-Keywords[ru]=akregator,configure,settings,archive,настроить,сконфигурировать,настройки,архив
 X-KDE-Keywords[pt_BR]=akregator,configurar,configurações,arquivar
 X-KDE-Keywords[sk]=akregator, nastaviť, nastavenia, archív
 X-KDE-Keywords[sr]=akregator,configure,settings,archive,Акрегатор,подешавање,поставке,архива
diff -urN kdepim-4.8.97/akregator/configuration/akregator_config_browser.desktop kdepim-4.8.97-patched/akregator/configuration/akregator_config_browser.desktop
--- kdepim-4.8.97/akregator/configuration/akregator_config_browser.desktop	2012-07-11 09:44:13.000000000 +1100
+++ kdepim-4.8.97-patched/akregator/configuration/akregator_config_browser.desktop	2012-07-16 21:46:55.756768011 +1100
@@ -130,6 +130,7 @@
 X-KDE-Keywords[nl]=akregator, instellen, instellingen, browser, web, intern
 X-KDE-Keywords[pl]=akregator, konfiguruj, ustawienia, przeglądarka, sieć, wewnętrzna
 X-KDE-Keywords[pt]=akregator, configurar, configuração, Navegador, Web, Interno
+X-KDE-Keywords[ru]=akregator, configure, settings, Browser, Web, Internal, настроить,сконфигурировать,настройки,браузер,веб,внутренние
 X-KDE-Keywords[pt_BR]=akregator,configurar,configurações,navegador,Internet,interno
 X-KDE-Keywords[sk]=akregator, nastaviť, nastavenia, prehliadač, web, interný
 X-KDE-Keywords[sr]=akregator,configure,settings,Browser,Web,Internal,Акрегатор,подешавање,поставке,прегледач,веб,унутрашњи
diff -urN kdepim-4.8.97/akregator/configuration/akregator_config_general.desktop kdepim-4.8.97-patched/akregator/configuration/akregator_config_general.desktop
--- kdepim-4.8.97/akregator/configuration/akregator_config_general.desktop	2012-07-11 09:44:13.000000000 +1100
+++ kdepim-4.8.97-patched/akregator/configuration/akregator_config_general.desktop	2012-07-16 21:46:55.756768011 +1100
@@ -151,6 +151,7 @@
 X-KDE-Keywords[nl]=akregator, instellen, instellingen, algemeen
 X-KDE-Keywords[pl]=akregator, konfiguruj, ustawienia, ogólne
 X-KDE-Keywords[pt]=akregator, configurar, configuração, Geral
+X-KDE-Keywords[ru]=akregator,configure,settings,General,настроить,сконфигурировать,настройки,главное
 X-KDE-Keywords[pt_BR]=akregator,configurar,configurações,geral
 X-KDE-Keywords[sk]=akregator, nastaviť, nastavenia, všeobecné
 X-KDE-Keywords[sr]=akregator,configure,settings,General,Акрегатор,подешавање,поставке,опште
diff -urN kdepim-4.8.97/akregator/plugins/onlinesync/akregator_config_onlinesync.desktop kdepim-4.8.97-patched/akregator/plugins/onlinesync/akregator_config_onlinesync.desktop
--- kdepim-4.8.97/akregator/plugins/onlinesync/akregator_config_onlinesync.desktop	2012-07-11 09:44:13.000000000 +1100
+++ kdepim-4.8.97-patched/akregator/plugins/onlinesync/akregator_config_onlinesync.desktop	2012-07-16 21:46:55.757768011 +1100
@@ -29,6 +29,7 @@
 X-KDE-Keywords[nl]=akregator, instellen, instellingen, online-lezers
 X-KDE-Keywords[pl]=akregator, konfiguruj, ustawienia, czytniki online
 X-KDE-Keywords[pt]=akregator, configurar, configuração, leitores online
+X-KDE-Keywords[ru]=akregator,configure,settings,online readers,настроить,сконфигурировать,настройки,дайджестов,онлайн,онлайновых дайджестов
 X-KDE-Keywords[pt_BR]=akregator,configurar,configurações,leitores online
 X-KDE-Keywords[sk]=akregator, nastaviť, nastavenia, online čítačky
 X-KDE-Keywords[sr]=akregator,configure,settings,online readers,Акрегатор,подешавање,поставке,читачи на вези
diff -urN kdepim-4.8.97/akregator/plugins/sharemicroblog/akregator_config_sharemicroblog.desktop kdepim-4.8.97-patched/akregator/plugins/sharemicroblog/akregator_config_sharemicroblog.desktop
--- kdepim-4.8.97/akregator/plugins/sharemicroblog/akregator_config_sharemicroblog.desktop	2012-07-11 09:44:13.000000000 +1100
+++ kdepim-4.8.97-patched/akregator/plugins/sharemicroblog/akregator_config_sharemicroblog.desktop	2012-07-16 21:46:55.757768011 +1100
@@ -29,6 +29,7 @@
 X-KDE-Keywords[nl]=akregator, instellen, instellingen, online-lezers
 X-KDE-Keywords[pl]=akregator, konfiguruj, ustawienia, czytniki online
 X-KDE-Keywords[pt]=akregator, configurar, configuração, leitores online
+X-KDE-Keywords[ru]=akregator,configure,settings,online readers,настроить,сконфигурировать,настройки,дайджестов,онлайн,онлайновых дайджестов
 X-KDE-Keywords[pt_BR]=akregator,configurar,configurações,leitores online
 X-KDE-Keywords[sk]=akregator, nastaviť, nastavenia, online čítačky
 X-KDE-Keywords[sr]=akregator,configure,settings,online readers,Акрегатор,подешавање,поставке,читачи на вези
diff -urN kdepim-4.8.97/akregator/plugins/sharemicroblog/akregator_sharemicroblog_plugin.desktop kdepim-4.8.97-patched/akregator/plugins/sharemicroblog/akregator_sharemicroblog_plugin.desktop
--- kdepim-4.8.97/akregator/plugins/sharemicroblog/akregator_sharemicroblog_plugin.desktop	2012-06-26 07:30:05.000000000 +1100
+++ kdepim-4.8.97-patched/akregator/plugins/sharemicroblog/akregator_sharemicroblog_plugin.desktop	2012-07-16 21:46:55.757768011 +1100
@@ -26,6 +26,7 @@
 Name[pl]=Udział artykułu online Akregatora
 Name[pt]=Partilha de Artigos 'Online' do Akregator
 Name[pt_BR]=Compartilhamento de artigos on-line do Akregator
+Name[ru]=Интернет-служба обмена статьями Akregator
 Name[sl]=Deljenje sestavkov prek spleta za Akregator
 Name[sr]=Дељење чланака на вези у Акрегатору
 Name[sr@ijekavian]=Дељење чланака на вези у Акрегатору
diff -urN kdepim-4.8.97/examples/coisceim/coisceimpart.desktop kdepim-4.8.97-patched/examples/coisceim/coisceimpart.desktop
--- kdepim-4.8.97/examples/coisceim/coisceimpart.desktop	2012-07-11 09:44:13.000000000 +1100
+++ kdepim-4.8.97-patched/examples/coisceim/coisceimpart.desktop	2012-07-16 21:46:55.758768011 +1100
@@ -23,6 +23,7 @@
 Name[nl]=CoisceimPart
 Name[pl]=Moduł Coisceim
 Name[pt]=CoisceimPart
+Name[ru]=CoisceimPart
 Name[pt_BR]=CoisceimPart
 Name[sk]=CoisceimPart
 Name[sl]=CoisceimPart
diff -urN kdepim-4.8.97/examples/coisceim/kontact-plugin/coisceim_plugin.desktop kdepim-4.8.97-patched/examples/coisceim/kontact-plugin/coisceim_plugin.desktop
--- kdepim-4.8.97/examples/coisceim/kontact-plugin/coisceim_plugin.desktop	2012-07-11 09:44:13.000000000 +1100
+++ kdepim-4.8.97-patched/examples/coisceim/kontact-plugin/coisceim_plugin.desktop	2012-07-16 21:46:55.758768011 +1100
@@ -40,6 +40,7 @@
 Comment[nl]=Coisceim-plug-in
 Comment[pl]=Wtyczka Coisceim
 Comment[pt]='Plugin' do Coisceim
+Comment[ru]=Модуль Coisceim
 Comment[pt_BR]=Plugin do Coisceim
 Comment[sk]=Plugin Coisceim
 Comment[sl]=Vstavek Coisceim
@@ -73,6 +74,7 @@
 Name[nb]=Trips
 Name[nds]=Reisen
 Name[nl]=Trips
+Name[ru]=Поездки
 Name[pl]=Podróże
 Name[pt]=Viagens
 Name[pt_BR]=Viagens
diff -urN kdepim-4.8.97/kjots/plasmoid/akonotes.desktop kdepim-4.8.97-patched/kjots/plasmoid/akonotes.desktop
--- kdepim-4.8.97/kjots/plasmoid/akonotes.desktop	2012-06-26 07:30:05.000000000 +1100
+++ kdepim-4.8.97-patched/kjots/plasmoid/akonotes.desktop	2012-07-16 21:46:55.759768011 +1100
@@ -26,6 +26,7 @@
 Name[pt]=Plasmóide de Akonotas
 Name[pt_BR]=Plasmoide Akonotas
 Name[ro]=Plasmoid Akonotes
+Name[ru]=Плазмоид Akonotes
 Name[sl]=Plazmoid Akonotes
 Name[sr]=Плазмоид Ако‑бележака
 Name[sr@ijekavian]=Плазмоид Ако‑биљежака
@@ -64,6 +65,7 @@
 Comment[pt]=Plasmóide de Akonotas
 Comment[pt_BR]=Plasmoide Akonotas
 Comment[ro]=Plasmoid Akonotes
+Comment[ru]=Плазмоид Akonotes
 Comment[sl]=Plazmoid Akonotes
 Comment[sr]=Плазмоид Ако‑бележака
 Comment[sr@ijekavian]=Плазмоид Ако‑биљежака
diff -urN kdepim-4.8.97/kleopatra/conf/kleopatra_config_appear.desktop kdepim-4.8.97-patched/kleopatra/conf/kleopatra_config_appear.desktop
--- kdepim-4.8.97/kleopatra/conf/kleopatra_config_appear.desktop	2012-07-11 09:44:13.000000000 +1100
+++ kdepim-4.8.97-patched/kleopatra/conf/kleopatra_config_appear.desktop	2012-07-16 21:46:55.775768011 +1100
@@ -172,6 +172,7 @@
 X-KDE-Keywords[nl]=kleur,lettertype, instellingen
 X-KDE-Keywords[pl]=kolor,czcionka,konfiguracja
 X-KDE-Keywords[pt]=cor,tipo de letra,configuração
+X-KDE-Keywords[ru]=color,font,configuration,цвет,шрифт,настройка,конфигурация
 X-KDE-Keywords[pt_BR]=cor,fonte, configuração
 X-KDE-Keywords[sk]=farba,písmo, konfigurácia
 X-KDE-Keywords[sr]=color,font,configuration,боја,фонт,постава,подешавање
diff -urN kdepim-4.8.97/kleopatra/conf/kleopatra_config_cryptooperations.desktop kdepim-4.8.97-patched/kleopatra/conf/kleopatra_config_cryptooperations.desktop
--- kdepim-4.8.97/kleopatra/conf/kleopatra_config_cryptooperations.desktop	2012-07-11 09:44:13.000000000 +1100
+++ kdepim-4.8.97-patched/kleopatra/conf/kleopatra_config_cryptooperations.desktop	2012-07-16 21:46:55.783768011 +1100
@@ -42,6 +42,7 @@
 Name[pt]=Operações de Criptografia
 Name[pt_BR]=Operações de criptografia
 Name[ro]=Operații criptografice
+Name[ru]=Операции шифрования
 Name[sl]=Operacije šifriranja
 Name[sr]=Шифрарски поступци
 Name[sr@ijekavian]=Шифрарски поступци
@@ -83,6 +84,7 @@
 Comment[pt]=Configuração das Operações de Criptografia
 Comment[pt_BR]=Configuração das operações de criptografia
 Comment[ro]=Configurarea operațiilor criptografice
+Comment[ru]=Настройка операций шифрования
 Comment[sl]=Nastavitev operacij šifriranja
 Comment[sr]=Подешавање шифрарских поступака
 Comment[sr@ijekavian]=Подешавање шифрарских поступака
@@ -113,6 +115,7 @@
 X-KDE-Keywords[nl]=ondertekenen,versleutelen,e-mail,snelle modus,controlesom,instellingen
 X-KDE-Keywords[pl]=podpisz,szyfruj,email,szybki tryb,suma kontrolna,konfiguracja
 X-KDE-Keywords[pt]=assinar,encriptar,e-mail,modo rápido,verificação,configuração
+X-KDE-Keywords[ru]=sign,encrypt,email,quick mode,checksum,configuration,подпись,шифрование,письмо,быстрый режим,контрольная сумма,настройка,конфигурация
 X-KDE-Keywords[pt_BR]=assinar,criptografar,e-mail,modo rápido,soma de verificação,configuração
 X-KDE-Keywords[sk]=podpísať,šifrovať,email,rýchly režim,kontrolný súčet,nastavenie
 X-KDE-Keywords[sr]=sign,encrypt,email,quick mode,checksum,configuration,потпис,шифровање,е‑пошта,брзи режим,контролна сума,постава,подешавање
diff -urN kdepim-4.8.97/kleopatra/conf/kleopatra_config_dirserv.desktop kdepim-4.8.97-patched/kleopatra/conf/kleopatra_config_dirserv.desktop
--- kdepim-4.8.97/kleopatra/conf/kleopatra_config_dirserv.desktop	2012-07-11 09:44:13.000000000 +1100
+++ kdepim-4.8.97-patched/kleopatra/conf/kleopatra_config_dirserv.desktop	2012-07-16 21:46:55.783768011 +1100
@@ -143,6 +143,7 @@
 X-KDE-Keywords[nl]=ldap,directory,services,hkp,keyserver
 X-KDE-Keywords[pl]=ldap,katalog,usługi,hkp,serwer kluczy
 X-KDE-Keywords[pt]=LDAP,directório,serviços,HKP,servidor de chaves
+X-KDE-Keywords[ru]=ldap,directory,services,hkp,keyserver,папка,директория,каталог,службы
 X-KDE-Keywords[pt_BR]=LDAP,diretório,serviços,hkp,servidor de chaves
 X-KDE-Keywords[sk]=ldap,adresár,služby,khp,server kľúčov
 X-KDE-Keywords[sr]=ldap,directory,services,hkp,keyserver,ЛДАП,именик,сервиси,ХКП,сервер кључева
diff -urN kdepim-4.8.97/kleopatra/conf/kleopatra_config_gnupgsystem.desktop kdepim-4.8.97-patched/kleopatra/conf/kleopatra_config_gnupgsystem.desktop
--- kdepim-4.8.97/kleopatra/conf/kleopatra_config_gnupgsystem.desktop	2012-07-11 09:44:13.000000000 +1100
+++ kdepim-4.8.97-patched/kleopatra/conf/kleopatra_config_gnupgsystem.desktop	2012-07-16 21:46:55.784768011 +1100
@@ -120,6 +120,7 @@
 X-KDE-Keywords[nb]=gpg,gpgsm,gpg-agent,dirmngr,scdaemon,kleopatra,signering,kryptering
 X-KDE-Keywords[nl]=gpg,gpgsm,gpg-agent,dirmngr,scdaemon,kleopatra,ondertekenen,versleuteling
 X-KDE-Keywords[pl]=gpg,gpgsm,gpg-agent,dirmngr,scdaemon,kleopatra,podpisywanie,szyfrowanie
+X-KDE-Keywords[ru]=gpg,gpgsm,gpg-agent,dirmngr,scdaemon,kleopatra,signing,encryption,gpg-агент,подписывание,шифрование
 X-KDE-Keywords[pt]=gpg,gpgsm,gpg-agent,dirmngr,scdaemon,kleopatra,assinatura,encriptação
 X-KDE-Keywords[pt_BR]=gpg,gpgsm,gpg-agent,dirmngr,scdaemon,kleopatra,assinando,criptografar
 X-KDE-Keywords[sk]=gpg,gpgsm,gpg-agent,dirmngr,scdaemon,kleopatra,podpisovanie,šifrovanie
diff -urN kdepim-4.8.97/kleopatra/conf/kleopatra_config_smimevalidation.desktop kdepim-4.8.97-patched/kleopatra/conf/kleopatra_config_smimevalidation.desktop
--- kdepim-4.8.97/kleopatra/conf/kleopatra_config_smimevalidation.desktop	2012-07-11 09:44:13.000000000 +1100
+++ kdepim-4.8.97-patched/kleopatra/conf/kleopatra_config_smimevalidation.desktop	2012-07-16 21:46:55.784768011 +1100
@@ -119,6 +119,7 @@
 X-KDE-Keywords[nb]=smime,validering,kleopatra,signering,kryptering
 X-KDE-Keywords[nl]=smime,validatie,kleopatra,ondertekenen,versleuteling
 X-KDE-Keywords[pl]=smime,walidacja,kleopatra,podpisywanie,szyfrowanie
+X-KDE-Keywords[ru]=smime,validation,kleopatra,signing,encryption,проверка,подписывание,шифрование
 X-KDE-Keywords[pt]=smime,validação,kleopatra,assinatura,encriptação
 X-KDE-Keywords[pt_BR]=smime,validação,kleopatra,assinando,criptografar
 X-KDE-Keywords[sk]=smime,overenie,kleopatra,podpisovanie,šifrovanie
diff -urN kdepim-4.8.97/kleopatra/kleopatra_signencryptfiles.desktop kdepim-4.8.97-patched/kleopatra/kleopatra_signencryptfiles.desktop
--- kdepim-4.8.97/kleopatra/kleopatra_signencryptfiles.desktop	2012-06-26 07:30:05.000000000 +1100
+++ kdepim-4.8.97-patched/kleopatra/kleopatra_signencryptfiles.desktop	2012-07-16 21:46:55.784768011 +1100
@@ -81,6 +81,7 @@
 Name[pt]=Assinar & Cifrar o Ficheiro
 Name[pt_BR]=Assinar e criptografar o arquivo
 Name[ro]=Semnează și criptează fișierul
+Name[ru]=Подписать и зашифровать файл
 Name[sl]=Podpiši in šifriraj datoteko
 Name[sr]=Потпиши и шифруј фајл
 Name[sr@ijekavian]=Потпиши и шифруј фајл
diff -urN kdepim-4.8.97/kleopatra/kleopatra_signencryptfolders.desktop kdepim-4.8.97-patched/kleopatra/kleopatra_signencryptfolders.desktop
--- kdepim-4.8.97/kleopatra/kleopatra_signencryptfolders.desktop	2012-06-28 06:39:33.000000000 +1100
+++ kdepim-4.8.97-patched/kleopatra/kleopatra_signencryptfolders.desktop	2012-07-16 21:46:55.785768011 +1100
@@ -75,6 +75,7 @@
 Name[pl]=Archiwizuj, podpisz i zaszyfruj katalog
 Name[pt]=Arquivar, Assinar && Encriptar a Pasta
 Name[pt_BR]=Arquivar, assinar e criptografar a pasta
+Name[ru]=Архивировать,подписать и зашифровать папку
 Name[sr]=Архивирај, потпиши и шифруј фасциклу
 Name[sr@ijekavian]=Архивирај, потпиши и шифруј фасциклу
 Name[sr@ijekavianlatin]=Arhiviraj, potpiši i šifruj fasciklu
@@ -113,6 +114,7 @@
 Name[pl]=Archiwizuj i zaszyfruj katalog
 Name[pt]=Arquivar && Cifrar a Pasta
 Name[pt_BR]=Arquivar e criptografar a pasta
+Name[ru]=Архивировать и зашифровать папку
 Name[sr]=Архивирај и шифруј фасциклу
 Name[sr@ijekavian]=Архивирај и шифруј фасциклу
 Name[sr@ijekavianlatin]=Arhiviraj i šifruj fasciklu
diff -urN kdepim-4.8.97/kmail/kcm_kpimidentities/kcm_kpimidentities.desktop kdepim-4.8.97-patched/kmail/kcm_kpimidentities/kcm_kpimidentities.desktop
--- kdepim-4.8.97/kmail/kcm_kpimidentities/kcm_kpimidentities.desktop	2012-07-11 09:44:13.000000000 +1100
+++ kdepim-4.8.97-patched/kmail/kcm_kpimidentities/kcm_kpimidentities.desktop	2012-07-16 21:46:55.785768011 +1100
@@ -165,6 +165,7 @@
 X-KDE-Keywords[nl]=kmail,identiteit
 X-KDE-Keywords[pl]=kmail,tożsamość
 X-KDE-Keywords[pt]=kmail,identidade
+X-KDE-Keywords[ru]=kmail,identity,профиль
 X-KDE-Keywords[pt_BR]=kmail,identidade
 X-KDE-Keywords[sk]=kmail,identita
 X-KDE-Keywords[sr]=kmail,identity,К‑пошта,идентитет
diff -urN kdepim-4.8.97/kmail/kmail_addattachmentservicemenu.desktop kdepim-4.8.97-patched/kmail/kmail_addattachmentservicemenu.desktop
--- kdepim-4.8.97/kmail/kmail_addattachmentservicemenu.desktop	2012-06-26 07:30:05.000000000 +1100
+++ kdepim-4.8.97-patched/kmail/kmail_addattachmentservicemenu.desktop	2012-07-16 21:46:55.785768011 +1100
@@ -30,6 +30,7 @@
 Name[pl]=Wyślij jako załącznik poczty
 Name[pt]=Enviar como Anexo do E-Mail
 Name[pt_BR]=Enviar como anexo de e-mail
+Name[ru]=Отправить как вложение в письмо
 Name[sr]=Пошаљи као прилог е‑поште
 Name[sr@ijekavian]=Пошаљи као прилог е‑поште
 Name[sr@ijekavianlatin]=Pošalji kao prilog e‑pošte
diff -urN kdepim-4.8.97/kmail/kmail_config_accounts.desktop kdepim-4.8.97-patched/kmail/kmail_config_accounts.desktop
--- kdepim-4.8.97/kmail/kmail_config_accounts.desktop	2012-07-11 09:44:13.000000000 +1100
+++ kdepim-4.8.97-patched/kmail/kmail_config_accounts.desktop	2012-07-16 21:46:55.786768011 +1100
@@ -168,6 +168,7 @@
 X-KDE-Keywords[nl]=kmail,accounts
 X-KDE-Keywords[pl]=kmail,konta
 X-KDE-Keywords[pt]=kmail,contas
+X-KDE-Keywords[ru]=kmail,accounts,учётные записи
 X-KDE-Keywords[pt_BR]=kmail,contas
 X-KDE-Keywords[sk]=kmail,účty
 X-KDE-Keywords[sr]=kmail,accounts,К‑пошта,налог
diff -urN kdepim-4.8.97/kmail/kmail_config_appearance.desktop kdepim-4.8.97-patched/kmail/kmail_config_appearance.desktop
--- kdepim-4.8.97/kmail/kmail_config_appearance.desktop	2012-07-11 09:44:13.000000000 +1100
+++ kdepim-4.8.97-patched/kmail/kmail_config_appearance.desktop	2012-07-16 21:46:55.786768011 +1100
@@ -171,6 +171,7 @@
 X-KDE-Keywords[nl]=kmail,uiterlijk
 X-KDE-Keywords[pl]=kmail,wygląd
 X-KDE-Keywords[pt]=kmail,aparência
+X-KDE-Keywords[ru]=kmail,appearance,внешний вид
 X-KDE-Keywords[pt_BR]=kmail,aparência
 X-KDE-Keywords[sk]=kmail,vzhľad
 X-KDE-Keywords[sr]=kmail,appearance,К‑пошта,изглед
diff -urN kdepim-4.8.97/kmail/kmail_config_composer.desktop kdepim-4.8.97-patched/kmail/kmail_config_composer.desktop
--- kdepim-4.8.97/kmail/kmail_config_composer.desktop	2012-07-11 09:44:13.000000000 +1100
+++ kdepim-4.8.97-patched/kmail/kmail_config_composer.desktop	2012-07-16 21:46:55.787768011 +1100
@@ -145,6 +145,7 @@
 X-KDE-Keywords[nl]=kmail,opsteller
 X-KDE-Keywords[pl]=kmail,edytor,kompozytor
 X-KDE-Keywords[pt]=kmail,compositor
+X-KDE-Keywords[ru]=kmail,composer,редактор писем
 X-KDE-Keywords[pt_BR]=kmail,compositor
 X-KDE-Keywords[sk]=kmail,editor
 X-KDE-Keywords[sr]=kmail,composer,К‑пошта,састављач
diff -urN kdepim-4.8.97/kmail/kmail_config_identity.desktop kdepim-4.8.97-patched/kmail/kmail_config_identity.desktop
--- kdepim-4.8.97/kmail/kmail_config_identity.desktop	2012-07-11 09:44:13.000000000 +1100
+++ kdepim-4.8.97-patched/kmail/kmail_config_identity.desktop	2012-07-16 21:46:55.787768011 +1100
@@ -169,6 +169,7 @@
 X-KDE-Keywords[nl]=kmail,identiteit
 X-KDE-Keywords[pl]=kmail,tożsamość
 X-KDE-Keywords[pt]=kmail,identidade
+X-KDE-Keywords[ru]=kmail,identity,профиль
 X-KDE-Keywords[pt_BR]=kmail,identidade
 X-KDE-Keywords[sk]=kmail,identita
 X-KDE-Keywords[sr]=kmail,identity,К‑пошта,идентитет
diff -urN kdepim-4.8.97/kmail/kmail_config_misc.desktop kdepim-4.8.97-patched/kmail/kmail_config_misc.desktop
--- kdepim-4.8.97/kmail/kmail_config_misc.desktop	2012-06-28 06:39:33.000000000 +1100
+++ kdepim-4.8.97-patched/kmail/kmail_config_misc.desktop	2012-07-16 21:46:55.787768011 +1100
@@ -164,6 +164,7 @@
 X-KDE-Keywords[nl]=kmail,overig
 X-KDE-Keywords[pl]=kmail,rożne
 X-KDE-Keywords[pt]=kmail,diversos
+X-KDE-Keywords[ru]=kmail,misc,прочее
 X-KDE-Keywords[pt_BR]=kmail,diversos
 X-KDE-Keywords[sk]=kmail,rôzne
 X-KDE-Keywords[sr]=kmail,misc,К‑пошта,разно
diff -urN kdepim-4.8.97/kmail/kmail_config_security.desktop kdepim-4.8.97-patched/kmail/kmail_config_security.desktop
--- kdepim-4.8.97/kmail/kmail_config_security.desktop	2012-07-11 09:44:13.000000000 +1100
+++ kdepim-4.8.97-patched/kmail/kmail_config_security.desktop	2012-07-16 21:46:55.788768011 +1100
@@ -172,6 +172,7 @@
 X-KDE-Keywords[nl]=kmail,beveiliging
 X-KDE-Keywords[pl]=kmail,bezpieczeństwo
 X-KDE-Keywords[pt]=kmail,segurança
+X-KDE-Keywords[ru]=kmail,security,безопасность
 X-KDE-Keywords[pt_BR]=kmail,segurança
 X-KDE-Keywords[sk]=kmail,bezpečnosť
 X-KDE-Keywords[sr]=kmail,security,К‑пошта,безбедност
diff -urN kdepim-4.8.97/kontact/plugins/kmail/kcmkmailsummary.desktop kdepim-4.8.97-patched/kontact/plugins/kmail/kcmkmailsummary.desktop
--- kdepim-4.8.97/kontact/plugins/kmail/kcmkmailsummary.desktop	2012-07-11 09:44:13.000000000 +1100
+++ kdepim-4.8.97-patched/kontact/plugins/kmail/kcmkmailsummary.desktop	2012-07-16 21:46:55.788768011 +1100
@@ -145,6 +145,7 @@
 X-KDE-Keywords[nl]=e-mail, samenvatting, instellen, instellingen
 X-KDE-Keywords[pl]=email, podsumowanie, konfiguruj, ustawienia
 X-KDE-Keywords[pt]=e-mail, resumo, configurar, configuração
+X-KDE-Keywords[ru]=email,summary,configure,settings,почта,электронная,дайджест,настроить,сконфигурировать,настройки
 X-KDE-Keywords[pt_BR]=e-mail,resumo,configurar,configurações
 X-KDE-Keywords[sk]=email, súhrn, nastaviť, nastavenia
 X-KDE-Keywords[sr]=email,summary,configure,settings,е‑пошта,сажета,подешавање,подесити,поставке
diff -urN kdepim-4.8.97/kontact/plugins/korganizer/kcmapptsummary.desktop kdepim-4.8.97-patched/kontact/plugins/korganizer/kcmapptsummary.desktop
--- kdepim-4.8.97/kontact/plugins/korganizer/kcmapptsummary.desktop	2012-07-11 09:44:13.000000000 +1100
+++ kdepim-4.8.97-patched/kontact/plugins/korganizer/kcmapptsummary.desktop	2012-07-16 21:46:55.788768011 +1100
@@ -128,6 +128,7 @@
 X-KDE-Keywords[nl]=agenda, afspraken, instellen, instellingen
 X-KDE-Keywords[pl]=kalendarz, wydarzenia, konfiguruj, ustawienia
 X-KDE-Keywords[pt]=calendário, eventos, configurar, configuração
+X-KDE-Keywords[ru]=calendar,events,configure,settings,календарь,события,настроить,сконфигурировать,настройки
 X-KDE-Keywords[pt_BR]=calendário,eventos,configurar,configurações
 X-KDE-Keywords[sk]=kalendár, udalosti, nastaviť, nastavenia
 X-KDE-Keywords[sr]=calendar,events,configure,settings,календар,догађаји,подешавање,подесити,поставке
diff -urN kdepim-4.8.97/kontact/plugins/korganizer/kcmtodosummary.desktop kdepim-4.8.97-patched/kontact/plugins/korganizer/kcmtodosummary.desktop
--- kdepim-4.8.97/kontact/plugins/korganizer/kcmtodosummary.desktop	2012-07-11 09:44:13.000000000 +1100
+++ kdepim-4.8.97-patched/kontact/plugins/korganizer/kcmtodosummary.desktop	2012-07-16 21:46:55.789768011 +1100
@@ -129,6 +129,7 @@
 X-KDE-Keywords[nl]=agenda, taken, instellen, instellingen
 X-KDE-Keywords[pl]=kalendarz, do zrobienia, konfiguruj, ustawienia
 X-KDE-Keywords[pt]=calendário,por-fazer,configurar,configuração
+X-KDE-Keywords[ru]=calendar,todos,configure,settings,календарь,задачи,настроить,сконфигурировать,настройки
 X-KDE-Keywords[pt_BR]=calendário,a fazeres,configurar,configuração
 X-KDE-Keywords[sk]=kalendár, úlohy, nastaviť, nastavenia
 X-KDE-Keywords[sr]=calendar,todos,configure,settings,календар,обавезе,подесити,подешавање,поставке
diff -urN kdepim-4.8.97/kontact/plugins/planner/kcmplanner.desktop kdepim-4.8.97-patched/kontact/plugins/planner/kcmplanner.desktop
--- kdepim-4.8.97/kontact/plugins/planner/kcmplanner.desktop	2012-07-11 09:44:13.000000000 +1100
+++ kdepim-4.8.97-patched/kontact/plugins/planner/kcmplanner.desktop	2012-07-16 21:46:55.790768011 +1100
@@ -126,6 +126,7 @@
 X-KDE-Keywords[km]=ប្រតិទិន កិច្ចការ​ត្រូវ​ធ្វើ កំណត់​រចនាសម្ព័ន្ធ កំណត់
 X-KDE-Keywords[nb]=kalender, gjøremål, oppsett. innstillinger
 X-KDE-Keywords[nl]=agenda, taken, instellen, instellingen
+X-KDE-Keywords[ru]=calendar,todos,configure,settings,календарь,задачи,настроить,сконфигурировать,настройки
 X-KDE-Keywords[pl]=kalendarz, do zrobienia, konfiguruj, ustawienia
 X-KDE-Keywords[pt]=calendário,por-fazer,configurar,configuração
 X-KDE-Keywords[pt_BR]=calendário,a fazeres,configurar,configuração
diff -urN kdepim-4.8.97/kontact/plugins/specialdates/kcmsdsummary.desktop kdepim-4.8.97-patched/kontact/plugins/specialdates/kcmsdsummary.desktop
--- kdepim-4.8.97/kontact/plugins/specialdates/kcmsdsummary.desktop	2012-07-11 09:44:13.000000000 +1100
+++ kdepim-4.8.97-patched/kontact/plugins/specialdates/kcmsdsummary.desktop	2012-07-16 21:46:55.790768011 +1100
@@ -130,6 +130,7 @@
 X-KDE-Keywords[nl]=verjaardag, trouwdatum, vakantie, instellen, instellingen
 X-KDE-Keywords[pl]=urodziny, rocznica, wakacje, święta, konfiguruj, ustawienia
 X-KDE-Keywords[pt]=nascimento,aniversário,feriado,configurar,configuração
+X-KDE-Keywords[ru]=birthday,anniversary,holiday,configure,settings,день рождения,годовщина,свадьбы,праздники,праздничные дни,настроить,сконфигурировать,настройки
 X-KDE-Keywords[pt_BR]=aniversário,feriado,configurar,configurações
 X-KDE-Keywords[sk]=narodeniny, výročie, sviatok, nastaviť, nastavenia
 X-KDE-Keywords[sr]=birthday,anniversary,holiday,configure,settings,рођендан,годишњица,празник,подесити,подешавање,поставке
diff -urN kdepim-4.8.97/kontact/src/kontactconfig.desktop kdepim-4.8.97-patched/kontact/src/kontactconfig.desktop
--- kdepim-4.8.97/kontact/src/kontactconfig.desktop	2012-06-26 07:30:05.000000000 +1100
+++ kdepim-4.8.97-patched/kontact/src/kontactconfig.desktop	2012-07-16 21:46:55.791768011 +1100
@@ -38,6 +38,7 @@
 Name[pl]=Konfiguracja Kontact
 Name[pt]=Configuração do Kontact
 Name[pt_BR]=Configuração do Kontact
+Name[ru]=Настройки Kontact
 Name[sl]=Nastavitve Kontacta
 Name[sr]=Подешавање Контакта
 Name[sr@ijekavian]=Подешавање Контакта
@@ -112,6 +113,7 @@
 X-KDE-Keywords[nl]=kontact
 X-KDE-Keywords[pl]=kontact
 X-KDE-Keywords[pt]=kontact
+X-KDE-Keywords[ru]=kontact
 X-KDE-Keywords[pt_BR]=kontact
 X-KDE-Keywords[sk]=kontakt
 X-KDE-Keywords[sr]=kontact,Контакт
diff -urN kdepim-4.8.97/korganizer/kcmconfigs/korganizer_configcolorsandfonts.desktop kdepim-4.8.97-patched/korganizer/kcmconfigs/korganizer_configcolorsandfonts.desktop
--- kdepim-4.8.97/korganizer/kcmconfigs/korganizer_configcolorsandfonts.desktop	2012-07-11 09:44:14.000000000 +1100
+++ kdepim-4.8.97-patched/korganizer/kcmconfigs/korganizer_configcolorsandfonts.desktop	2012-07-16 21:46:55.804768011 +1100
@@ -127,6 +127,7 @@
 X-KDE-Keywords[nb]=korganizer,farger,skrifter
 X-KDE-Keywords[nl]=korganizer,kleuren,lettertypen
 X-KDE-Keywords[pl]=korganizer,kolory,czcionki
+X-KDE-Keywords[ru]=korganizer,colors,fonts,цвета,шрифты
 X-KDE-Keywords[pt]=korganizer,cores,tipos de letra
 X-KDE-Keywords[pt_BR]=korganizer,cores,fontes
 X-KDE-Keywords[sk]=korganizer,farby,písma
diff -urN kdepim-4.8.97/korganizer/kcmconfigs/korganizer_configdesignerfields.desktop kdepim-4.8.97-patched/korganizer/kcmconfigs/korganizer_configdesignerfields.desktop
--- kdepim-4.8.97/korganizer/kcmconfigs/korganizer_configdesignerfields.desktop	2012-07-11 09:44:14.000000000 +1100
+++ kdepim-4.8.97-patched/korganizer/kcmconfigs/korganizer_configdesignerfields.desktop	2012-07-16 21:46:55.804768011 +1100
@@ -161,6 +161,7 @@
 X-KDE-Keywords[nl]=korganizer, instellen, instellingen, eigen velden
 X-KDE-Keywords[pl]=korganizer, konfiguruj, ustawienia, własne pola
 X-KDE-Keywords[pt]=korganizer, configurar, configuração, campos personalizados
+X-KDE-Keywords[ru]=korganizer,configure,settings,custom fields,настроить,сконфигурировать,настройки,дополнительные поля
 X-KDE-Keywords[pt_BR]=korganizer,configurar,configurações,campos personalizados
 X-KDE-Keywords[sk]=korganizer, nastaviť, nastavenia, vlastné polia
 X-KDE-Keywords[sr]=korganizer,configure,settings,custom fields,К‑организатор,подешавање,подесити,поставке,посебна поља
diff -urN kdepim-4.8.97/korganizer/kcmconfigs/korganizer_configfreebusy.desktop kdepim-4.8.97-patched/korganizer/kcmconfigs/korganizer_configfreebusy.desktop
--- kdepim-4.8.97/korganizer/kcmconfigs/korganizer_configfreebusy.desktop	2012-07-11 09:44:14.000000000 +1100
+++ kdepim-4.8.97-patched/korganizer/kcmconfigs/korganizer_configfreebusy.desktop	2012-07-16 21:46:55.804768011 +1100
@@ -157,6 +157,7 @@
 X-KDE-Keywords[nl]=korganizer,vrijbezet,plannen
 X-KDE-Keywords[pl]=korganizer,wolnyzajęty,planowanie
 X-KDE-Keywords[pt]=korganizer,livre-ocupado,agendamento
+X-KDE-Keywords[ru]=korganizer,freebusy,scheduling,занятое время,расписание,планирование
 X-KDE-Keywords[pt_BR]=korganizer,livre-ocupado,agendamento
 X-KDE-Keywords[sk]=korganizer,freebusy,plánovanie
 X-KDE-Keywords[sr]=korganizer,freebusy,scheduling,К‑организатор,слободно-заузето,распоређивање
diff -urN kdepim-4.8.97/korganizer/kcmconfigs/korganizer_configgroupscheduling.desktop kdepim-4.8.97-patched/korganizer/kcmconfigs/korganizer_configgroupscheduling.desktop
--- kdepim-4.8.97/korganizer/kcmconfigs/korganizer_configgroupscheduling.desktop	2012-07-11 09:44:14.000000000 +1100
+++ kdepim-4.8.97-patched/korganizer/kcmconfigs/korganizer_configgroupscheduling.desktop	2012-07-16 21:46:55.805768011 +1100
@@ -156,6 +156,7 @@
 X-KDE-Keywords[nl]=korganizer,groep,plannen
 X-KDE-Keywords[pl]=korganizer,grupowanie,planowanie
 X-KDE-Keywords[pt]=korganizer,agendamento,escalonamento,grupo
+X-KDE-Keywords[ru]=korganizer,group,scheduling,группа,планирование,расписание
 X-KDE-Keywords[pt_BR]=korganizer,agendamento,escalonamento,grupo
 X-KDE-Keywords[sk]=korganizer,skupina,plánovanie
 X-KDE-Keywords[sr]=korganizer,group,scheduling,К‑организатор,група,груписање,распоред,распоређивање
diff -urN kdepim-4.8.97/korganizer/kcmconfigs/korganizer_configmain.desktop kdepim-4.8.97-patched/korganizer/kcmconfigs/korganizer_configmain.desktop
--- kdepim-4.8.97/korganizer/kcmconfigs/korganizer_configmain.desktop	2012-07-11 09:44:14.000000000 +1100
+++ kdepim-4.8.97-patched/korganizer/kcmconfigs/korganizer_configmain.desktop	2012-07-16 21:46:55.805768011 +1100
@@ -169,6 +169,7 @@
 X-KDE-Keywords[nl]=korganizer,hoofdpagina
 X-KDE-Keywords[pl]=korganizer,główny
 X-KDE-Keywords[pt]=korganizer,principal
+X-KDE-Keywords[ru]=korganizer,main,общее,главное,основное
 X-KDE-Keywords[pt_BR]=korganizer,principal
 X-KDE-Keywords[sk]=korganizer,hlavné
 X-KDE-Keywords[sr]=korganizer,main,К‑организатор,главно
diff -urN kdepim-4.8.97/korganizer/kcmconfigs/korganizer_configplugins.desktop kdepim-4.8.97-patched/korganizer/kcmconfigs/korganizer_configplugins.desktop
--- kdepim-4.8.97/korganizer/kcmconfigs/korganizer_configplugins.desktop	2012-06-26 07:30:05.000000000 +1100
+++ kdepim-4.8.97-patched/korganizer/kcmconfigs/korganizer_configplugins.desktop	2012-07-16 21:46:55.805768011 +1100
@@ -162,6 +162,7 @@
 X-KDE-Keywords[nl]=korganizer,plug-in,module
 X-KDE-Keywords[pl]=korganizer,wtyczka,moduł
 X-KDE-Keywords[pt]=korganizer,'plugin',módulo
+X-KDE-Keywords[ru]=korganizer,plugin,module,модуль,плагин
 X-KDE-Keywords[pt_BR]=korganizer,plugin,módulo
 X-KDE-Keywords[sk]=korganizer,plugin,modul
 X-KDE-Keywords[sr]=korganizer,plugin,module,К‑организатор,прикључак,модул
diff -urN kdepim-4.8.97/korganizer/kcmconfigs/korganizer_configtime.desktop kdepim-4.8.97-patched/korganizer/kcmconfigs/korganizer_configtime.desktop
--- kdepim-4.8.97/korganizer/kcmconfigs/korganizer_configtime.desktop	2012-07-11 09:44:14.000000000 +1100
+++ kdepim-4.8.97-patched/korganizer/kcmconfigs/korganizer_configtime.desktop	2012-07-16 21:46:55.806768011 +1100
@@ -133,6 +133,7 @@
 X-KDE-Keywords[nb]=korganizer,tid
 X-KDE-Keywords[nl]=korganizer,tijd
 X-KDE-Keywords[pl]=korganizer,czas
+X-KDE-Keywords[ru]=korganizer,time,время,дата
 X-KDE-Keywords[pt]=korganizer,hora
 X-KDE-Keywords[pt_BR]=korganizer,tempo
 X-KDE-Keywords[sk]=korganizer,čas
diff -urN kdepim-4.8.97/korganizer/kcmconfigs/korganizer_configviews.desktop kdepim-4.8.97-patched/korganizer/kcmconfigs/korganizer_configviews.desktop
--- kdepim-4.8.97/korganizer/kcmconfigs/korganizer_configviews.desktop	2012-07-11 09:44:14.000000000 +1100
+++ kdepim-4.8.97-patched/korganizer/kcmconfigs/korganizer_configviews.desktop	2012-07-16 21:46:55.806768011 +1100
@@ -166,6 +166,7 @@
 X-KDE-Keywords[nl]=korganizer,beeld
 X-KDE-Keywords[pl]=korganizer,widok
 X-KDE-Keywords[pt]=korganizer,ver
+X-KDE-Keywords[ru]=korganizer,view,представление
 X-KDE-Keywords[pt_BR]=korganizer,vsualizar
 X-KDE-Keywords[sk]=korganizer,zobraziť
 X-KDE-Keywords[sr]=korganizer,view,К‑организатор,приказ
diff -urN kdepim-4.8.97/korganizer/korgac/korgacagent.desktop kdepim-4.8.97-patched/korganizer/korgac/korgacagent.desktop
--- kdepim-4.8.97/korganizer/korgac/korgacagent.desktop	2012-06-26 07:30:05.000000000 +1100
+++ kdepim-4.8.97-patched/korganizer/korgac/korgacagent.desktop	2012-07-16 21:46:55.806768011 +1100
@@ -24,6 +24,7 @@
 Name[nb]=Påminnelsesnisse
 Name[nds]=Anstötendämoon
 Name[nl]=Herinneringsdaemon
+Name[ru]=Демон напоминаний
 Name[pa]=ਰੀਮਾਈਡਰ ਡੈਮਨ
 Name[pl]=Demon przypominania
 Name[pt]=Servidor de Chamadas de Atenção
@@ -64,6 +65,7 @@
 Comment[pl]=Demon przypominania zdarzeń i zadań
 Comment[pt]=Servidor de chamada de atenção dos eventos e tarefas
 Comment[pt_BR]=Servidor de lembretes dos eventos e tarefas
+Comment[ru]=Демон напоминаний о событиях и задачах
 Comment[sl]=Pritajeni program za opomnike glede dogodkov in opravil
 Comment[sr]=Демон за подсећање на догађаје и задатке
 Comment[sr@ijekavian]=Демон за подсјећање на догађаје и задатке
diff -urN kdepim-4.8.97/ktimetracker/ktimetracker_config_behavior.desktop kdepim-4.8.97-patched/ktimetracker/ktimetracker_config_behavior.desktop
--- kdepim-4.8.97/ktimetracker/ktimetracker_config_behavior.desktop	2012-06-26 07:30:05.000000000 +1100
+++ kdepim-4.8.97-patched/ktimetracker/ktimetracker_config_behavior.desktop	2012-07-16 21:46:55.807768011 +1100
@@ -127,6 +127,7 @@
 X-KDE-Keywords[pl]=ktimetracker, konfiguruj, ustawienia
 X-KDE-Keywords[pt]=ktimetracker, configurar, configuração
 X-KDE-Keywords[pt_BR]=ktimetracker,configurar,configurações
+X-KDE-Keywords[ru]=ktimetracker,configure,settings,настроить,сконфигурировать,настройки
 X-KDE-Keywords[sr]=ktimetracker,configure,settings,К‑пратилац-времена,подешавање,подесити,поставке
 X-KDE-Keywords[sr@ijekavian]=ktimetracker,configure,settings,К‑пратилац-времена,подешавање,подесити,поставке
 X-KDE-Keywords[sr@ijekavianlatin]=ktimetracker,configure,settings,K‑pratilac-vremena,podešavanje,podesiti,postavke
diff -urN kdepim-4.8.97/ktimetracker/ktimetracker_config_display.desktop kdepim-4.8.97-patched/ktimetracker/ktimetracker_config_display.desktop
--- kdepim-4.8.97/ktimetracker/ktimetracker_config_display.desktop	2012-06-26 07:30:05.000000000 +1100
+++ kdepim-4.8.97-patched/ktimetracker/ktimetracker_config_display.desktop	2012-07-16 21:46:55.807768011 +1100
@@ -152,6 +152,7 @@
 X-KDE-Keywords[pl]=ktimetracker, konfiguruj, ustawienia
 X-KDE-Keywords[pt]=ktimetracker, configurar, configuração
 X-KDE-Keywords[pt_BR]=ktimetracker,configurar,configurações
+X-KDE-Keywords[ru]=ktimetracker,configure,settings,настроить,сконфигурировать,настройки
 X-KDE-Keywords[sr]=ktimetracker,configure,settings,К‑пратилац-времена,подешавање,подесити,поставке
 X-KDE-Keywords[sr@ijekavian]=ktimetracker,configure,settings,К‑пратилац-времена,подешавање,подесити,поставке
 X-KDE-Keywords[sr@ijekavianlatin]=ktimetracker,configure,settings,K‑pratilac-vremena,podešavanje,podesiti,postavke
diff -urN kdepim-4.8.97/ktimetracker/ktimetracker_config_storage.desktop kdepim-4.8.97-patched/ktimetracker/ktimetracker_config_storage.desktop
--- kdepim-4.8.97/ktimetracker/ktimetracker_config_storage.desktop	2012-06-26 07:30:05.000000000 +1100
+++ kdepim-4.8.97-patched/ktimetracker/ktimetracker_config_storage.desktop	2012-07-16 21:46:55.807768011 +1100
@@ -126,6 +126,7 @@
 X-KDE-Keywords[pl]=ktimetracker, konfiguruj, ustawienia
 X-KDE-Keywords[pt]=ktimetracker, configurar, configuração
 X-KDE-Keywords[pt_BR]=ktimetracker,configurar,configurações
+X-KDE-Keywords[ru]=ktimetracker,configure,settings,настроить,сконфигурировать,настройки
 X-KDE-Keywords[sr]=ktimetracker,configure,settings,К‑пратилац-времена,подешавање,подесити,поставке
 X-KDE-Keywords[sr@ijekavian]=ktimetracker,configure,settings,К‑пратилац-времена,подешавање,подесити,поставке
 X-KDE-Keywords[sr@ijekavianlatin]=ktimetracker,configure,settings,K‑pratilac-vremena,podešavanje,podesiti,postavke
diff -urN kdepim-4.8.97/libkdepim/ldap/kcmldap.desktop kdepim-4.8.97-patched/libkdepim/ldap/kcmldap.desktop
--- kdepim-4.8.97/libkdepim/ldap/kcmldap.desktop	2012-07-11 09:44:14.000000000 +1100
+++ kdepim-4.8.97-patched/libkdepim/ldap/kcmldap.desktop	2012-07-16 21:46:55.808768011 +1100
@@ -40,6 +40,7 @@
 Name[pl]=Ustawienia serwera LDAP
 Name[pt]=Configuração do Servidor de LDAP
 Name[pt_BR]=Configurações do servidor LDAP
+Name[ru]=Настройки сервера LDAP
 Name[sk]=Nastavenia servera LDAP
 Name[sl]=Nastavitve strežnika LDAP
 Name[sr]=Поставке ЛДАП сервера
@@ -83,6 +84,7 @@
 Comment[pl]=Konfiguracja dostępnych serwerów LDAP
 Comment[pt]=Configurar os servidores de LDAP disponíveis
 Comment[pt_BR]=Configura os servidores LDAP disponíveis
+Comment[ru]=Настройка доступных серверов LDAP
 Comment[sl]=Nastavite razpoložljive strežnike LDAP
 Comment[sr]=Подешавање доступних ЛДАП сервера
 Comment[sr@ijekavian]=Подешавање доступних ЛДАП сервера
@@ -115,6 +117,7 @@
 X-KDE-Keywords[pl]=kaddressbook, konfiguruj, ustawienia, LDAP
 X-KDE-Keywords[pt]=kaddressbook, configurar, configuração, LDAP
 X-KDE-Keywords[pt_BR]=kaddressbook,configurar,configurações,LDAP
+X-KDE-Keywords[ru]=kaddressbook,configure,settings,LDAP,настроить,сконфигурировать,настройки
 X-KDE-Keywords[sr]=kaddressbook,configure,settings,LDAP,К‑адресар,подешавање,подесити,поставке,ЛДАП
 X-KDE-Keywords[sr@ijekavian]=kaddressbook,configure,settings,LDAP,К‑адресар,подешавање,подесити,поставке,ЛДАП
 X-KDE-Keywords[sr@ijekavianlatin]=kaddressbook,configure,settings,LDAP,K‑adresar,podešavanje,podesiti,postavke,LDAP
diff -urN kdepim-4.8.97/libkleo/libkleopatrarc.desktop kdepim-4.8.97-patched/libkleo/libkleopatrarc.desktop
--- kdepim-4.8.97/libkleo/libkleopatrarc.desktop	2012-07-11 09:44:14.000000000 +1100
+++ kdepim-4.8.97-patched/libkleo/libkleopatrarc.desktop	2012-07-16 21:46:55.809768011 +1100
@@ -33,6 +33,7 @@
 Name[pl]=TAR (kompatybilny z PGP®)
 Name[pt]=TAR (compatível com o PGP®)
 Name[pt_BR]=TAR (compatível com PGP®)
+Name[ru]=TAR (PGP®-совместимый)
 Name[sk]=TAR (PGP® kompatibilné)
 Name[sl]=TAR (združljiv s PGP)
 Name[sr]=тар (ПГП®-сагласно)
@@ -122,6 +123,7 @@
 Name[pa]=TAR (bzip2 ਕੰਪਰੈਸ਼ਨ ਨਾਲ)
 Name[pl]=TAR (z kompresją  bzip2)
 Name[pt]=TAR (com compressão bzip2)
+Name[ru]=TAR (со сжатием bzip2)
 Name[pt_BR]=TAR (compressão com bzip2)
 Name[sk]=TAR (s bzip2 kompresiou)
 Name[sl]=TAR (s stiskanjem bzip2)
@@ -687,6 +689,7 @@
 Name[pl]=Klucz na karcie inteligentnej
 Name[pt]=Chave do 'Smartcard'
 Name[pt_BR]=Chave do Smartcard
+Name[ru]=Ключ смарт карты
 Name[sl]=Ključ na pametni kartici
 Name[sr]=Кључ смарт-картица
 Name[sr@ijekavian]=Кључ смарт-картица
diff -urN kdepim-4.8.97/libkleo/libkleopatrarc-win32.desktop kdepim-4.8.97-patched/libkleo/libkleopatrarc-win32.desktop
--- kdepim-4.8.97/libkleo/libkleopatrarc-win32.desktop	2012-07-11 09:44:14.000000000 +1100
+++ kdepim-4.8.97-patched/libkleo/libkleopatrarc-win32.desktop	2012-07-16 21:46:55.809768011 +1100
@@ -33,6 +33,7 @@
 Name[pl]=TAR (kompatybilny z PGP®)
 Name[pt]=TAR (compatível com o PGP®)
 Name[pt_BR]=TAR (compatível com PGP®)
+Name[ru]=TAR (PGP®-совместимый)
 Name[sk]=TAR (PGP® kompatibilné)
 Name[sl]=TAR (združljiv s PGP)
 Name[sr]=тар (ПГП®-сагласно)
@@ -129,6 +130,7 @@
 Name[nl]=sha256sum
 Name[pa]=sha256sum
 Name[pl]=sha256sum
+Name[ru]=sha256sum
 Name[pt]=sha256sum
 Name[pt_BR]=sha256sum
 Name[sk]=sha256sum
@@ -646,6 +648,7 @@
 Name[pl]=Klucz na karcie inteligentnej
 Name[pt]=Chave do 'Smartcard'
 Name[pt_BR]=Chave do Smartcard
+Name[ru]=Ключ карты памяти
 Name[sl]=Ključ na pametni kartici
 Name[sr]=Кључ смарт-картица
 Name[sr@ijekavian]=Кључ смарт-картица
diff -urN kdepim-4.8.97/mailfilteragent/mailfilteragent.desktop kdepim-4.8.97-patched/mailfilteragent/mailfilteragent.desktop
--- kdepim-4.8.97/mailfilteragent/mailfilteragent.desktop	2012-07-11 09:44:14.000000000 +1100
+++ kdepim-4.8.97-patched/mailfilteragent/mailfilteragent.desktop	2012-07-16 21:46:55.810768011 +1100
@@ -20,6 +20,7 @@
 Name[pl]=Agent filtrowania poczty
 Name[pt]=Agente de Filtragem de E-mail
 Name[pt_BR]=Agente de filtros de e-mail
+Name[ru]=Агент почтового фильтра
 Name[sr]=Агент филтрирања поште
 Name[sr@ijekavian]=Агент филтрирања поште
 Name[sr@ijekavianlatin]=Agent filtriranja pošte
@@ -50,6 +51,7 @@
 Comment[pl]=Rozszerzenie do filtrowania poczty
 Comment[pt]=Extensão para filtrar as mensagens de e-mail
 Comment[pt_BR]=Extensão para filtros de e-mails
+Comment[ru]=Расширение для фильтрации электронных писем
 Comment[sr]=Проширење за филтрирање е‑поште
 Comment[sr@ijekavian]=Проширење за филтрирање е‑поште
 Comment[sr@ijekavianlatin]=Proširenje za filtriranje e‑pošte
diff -urN kdepim-4.8.97/mobile/tasks/tasks-mobile.desktop kdepim-4.8.97-patched/mobile/tasks/tasks-mobile.desktop
--- kdepim-4.8.97/mobile/tasks/tasks-mobile.desktop	2012-06-26 07:30:06.000000000 +1100
+++ kdepim-4.8.97-patched/mobile/tasks/tasks-mobile.desktop	2012-07-16 21:46:55.810768011 +1100
@@ -23,6 +23,7 @@
 Name[nl]=Taken
 Name[pl]=Zadania
 Name[pt]=Tarefas
+Name[ru]=Задачи
 Name[pt_BR]=Tarefas
 Name[sk]=Úlohy
 Name[sr]=Задаци
diff -urN kdepim-4.8.97/plugins/messageviewer/bodypartformatter/application_ms-tnef.desktop kdepim-4.8.97-patched/plugins/messageviewer/bodypartformatter/application_ms-tnef.desktop
--- kdepim-4.8.97/plugins/messageviewer/bodypartformatter/application_ms-tnef.desktop	2012-06-26 07:30:06.000000000 +1100
+++ kdepim-4.8.97-patched/plugins/messageviewer/bodypartformatter/application_ms-tnef.desktop	2012-07-16 21:46:55.811768011 +1100
@@ -26,6 +26,7 @@
 Name[pl]=TNEF
 Name[pt]=TNEF
 Name[pt_BR]=TNEF
+Name[ru]=TNEF
 Name[sk]=TNEF
 Name[sl]=TNEF
 Name[sr]=ТНЕФ
@@ -60,6 +61,7 @@
 Comment[nds]=En Hööftdeel-Formateermoduul för TNEF-Bilagen
 Comment[nl]=Een opmaakplug-in voor TNEF-bijlagen
 Comment[pl]=Wtyczka formatująca treść dla załączników TNEF
+Comment[ru]=Модуль для форматирования текста для вложений TNEF
 Comment[pt]=Um 'plugin' de formatação para os anexos TNEF
 Comment[pt_BR]=Um plugin formatador de componente para anexos TNEF
 Comment[sl]=Oblikovalni vstavek za priloge TNEF
